Solaris specific patch to use XPG7 standards.
NOTE : This patch will not be pushed upstream, as community is using
       XPG5 standards.

--- ghostscript-9.22/configure.ac.orig	2017-12-05 23:14:42.770052878 +0000
+++ ghostscript-9.22/configure.ac	2017-12-05 23:15:06.051469780 +0000
@@ -654,7 +654,7 @@
 
 if test "x$HAVE_PREAD_PWRITE" != "x"; then
   save_cflags=$CFLAGS
-  CFLAGS="$CFLAGS -D__USE_UNIX98=1 -D_XOPEN_SOURCE=500"
+  CFLAGS="$CFLAGS -D__USE_UNIX98=1 -D_XOPEN_SOURCE=700"
   AC_CHECK_DECLS([pwrite,pread], [], [HAVE_PREAD_PWRITE=])
   CFLAGS=$save_cflags
   if test "x$HAVE_PREAD_PWRITE" != "x"; then
